core/README.md

```diff
@@ -20,3 +20,8 @@ The process of adding new collaterals depends on the token type used. This is du
         - The file should be named using the name from `1.`
         - The file should export `CalleeFunctions`
         - The file should be imported in the [`calleeFunctions/index.ts`](./src/calleeFunctions/index.ts)
+
+## Environment Variables
+
+`LOCAL_RPC_URL` - link to the blockchiain RPC the tests will interact with, defaults to `localhost:8545`
+`REMOTE_RPC_URL` - linkt to the blockchain RPC where hardhat fork will copy data from, no default value - it has to be provided.
```

Comment:
Please follow the format of the existing sections in other services:
- [ ] should be a list
- [ ] should not contain typos
- [ ] should specify exact default values
- [ ] should specify required or optional
- [ ] sorted so required to be on top of the list
```suggestion
- `REMOTE_RPC_URL`: (required for tests, simulations) Ethereum RPC url with access to historical data. Used by hardhat to fetch relevant data
- `LOCAL_RPC_URL`: (optional, default `http://localhost:8545`) Hardhat RPC url. Used by tests and simulations
```